src/auth/identitypoolclient.ts

```diff
@@ -36,6 +36,10 @@ interface SubjectTokenJsonResponse {
   [key: string]: string;
 }
 
+/** The workforce audience pattern. */
+const WORKFORCE_AUDIENCE_PATTERN =
+  '//iam.googleapis.com/projects/.+/locations/.+/workforcePools/.+/providers/.+';
```

Comment:
//iam.googleapis.com/locations/.+/workforcePools/.+/providers/.+

Reply:
I only saw this format in the old design [doc](https://docs.google.com/document/d/1PFz90avGF0w6Vb-8macSESmVj_du2MUgvW3YM9PDxtk/edit), not sure that was outdated, was that the reference?